src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -303,7 +313,34 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionContext context)
     }
   }
 
-  /** conversion utility for protobuf Timestamp difference to java.time.Duration */
+  private static void profileAccounting(ExecutedActionMetadata executedActionMetadata) {
+    MillisSinceEpochToNanosConverter converter = BlazeClock.createMillisSinceEpochToNanosConverter();
+
+    logProfileTask(converter, executedActionMetadata.getQueuedTimestamp(),
+        executedActionMetadata.getWorkerStartTimestamp(),
+        REMOTE_QUEUE, "Remote execution queuing time");
```

Comment:
Consider using just `REMOTE_QUEUE.toString()` here (and similar below) instead of the full description. I think that the enum names are still easy to read for humans and much shorter, so you don't see abbreviations like `Remote execu...` like you do now.

Reply:
Also consider removing `Remote ...` prefix as they are all under `execute remotely` span.

Reply:
Done.